rmw_opensplice_cpp/CMakeLists.txt

```diff
@@ -41,6 +41,11 @@ endif()
 ament_package()
 
 install(
+  DIRECTORY include/
+  DESTINATION include
+)
```

Comment:
I think I don't really understand what `ament_export_include_directories` does. In this case, I'm exporting `include/${PROJECT_NAME}/impl` because I need `rclcpp` to be able to include `rmw/get_type_support_handle.h`, but I don't need any other package to be able to include the full include path (i.e. `rmw_opensplice_cpp/impl/rmw/get_type_support_handle.h`).

However I still need to install the `include/` directory, or the downstream packages can't find the header files. Do I have to add `include` to `ament_export_include_directories` even if I don't want it to be exported?